deployment/common/changeset/deploy_link_token.go

```diff
@@ -10,20 +10,24 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/shared/generated/link_token"
 )
 
-var _ deployment.ChangeSet[uint64] = DeployLinkToken
+var _ deployment.ChangeSet[[]uint64] = DeployLinkToken
```

Comment:
@ecPablo Just more flexible this way (especially for tests)

Reply:
as a general rule I think changeset config being multi-chain is always useful even if we only go 1 chain at time in prod

Reply:
good to know!